.github/workflows/build.yaml

```diff
@@ -283,7 +283,7 @@ jobs:
           node-version: '18'
       - name: Generate Bicep extensibility types from OpenAPI specs
         run: |
-          make generate-bicep-types VERSION=${{ env.REL_VERSION == 'edge' && 'latest' || env.REL_VERSION }}
+          make generate-bicep-types VERSION=${{ env.REL_CHANNEL == 'edge' && 'latest' || env.REL_CHANNEL }}
```

Comment:
was this wrong before? if not why change over to `REL_CHANNEL`?

Reply:
yea it was, we pull from the version channel in the bicep config so the uploads should also be to the channel tag